clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1643,8 +1676,10 @@ event_kernel_xfer(void *drcontext, const dr_kernel_xfer_info_t *info)
     }
     BUF_PTR(data->seg_base) +=
         instru->append_marker(BUF_PTR(data->seg_base), marker_type, marker_val);
-    if (file_ops_func.handoff_buf == NULL)
-        process_and_output_buffer(drcontext, false);
+    // Append a timestamp.
```

Comment:
"Append a timestamp" doesn't say much more than the `append_timestamp` call in the next line. Probably it should say why this timestamp is useful.

Reply:
Done.